src/mainboard/pcengines/apu2/romstage.c

```diff
@@ -71,8 +78,21 @@ void cache_as_ram_main(unsigned long bist, unsigned long cpu_init_detectedx)
 
 		hudson_clk_output_48Mhz();
 		post_code(0x31);
-		console_init();
 
+		dev = PCI_DEV(0, 0x14, 3);
+		data = pci_read_config32(dev, LPC_IO_OR_MEM_DECODE_ENABLE);
+		/* enable 0x2e/0x4e IO decoding before configuring SuperIO */
+		pci_write_config32(dev, LPC_IO_OR_MEM_DECODE_ENABLE, data | 3);
+
+		/* COM2 on apu5 is reserved so only COM1 should be supported */
+		if ((CONFIG_UART_FOR_CONSOLE == 1) &&
+			 !IS_ENABLED(CONFIG_BOARD_PCENGINES_APU5))
+			nuvoton_enable_serial(SERIAL2_DEV, CONFIG_TTYS0_BASE);
```

Comment:
@miczyg1 this is not K&R alignment or something that comply to coreboot standards.

Reply:
@pietrushnic good point. I have removed the whitespace there.

Weird that lint did not catch it before sending patch to coreboot.